packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -700,4 +700,28 @@
     <label>Rechercher à la place {{0}}</label>
     <name>quantic_SearchInsteadFor</name>
   </customLabels>
+  <customLabels>
+    <label>Non pertinent</label>
+    <name>quantic_Irrelevant</name>
+  </customLabels>
+  <customLabels>
+    <label>pas précis</label>
+    <name>quantic_NotAccurate</name>
+  </customLabels>
+  <customLabels>
+    <label>Pas à jour</label>
+    <name>quantic_OutOfDate</name>
+  </customLabels>
+  <customLabels>
+    <label>Nocif</label>
```

Comment:
@mmitiche @SimonMilord As we're talking about "réponse", let's make it feminine: Nocive

Reply:
Nuisible? Nefaste?